proto/RobotCommands.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package proto;
+
+message RobotCommand {
+    uint32 id = 1;
+
+    float velocity_x = 2;
+    float velocity_y = 3;
+    float angle = 4;
+    float angular_velocity = 5;
+    bool use_angular_velocity = 6;
```

Comment:
You don't need this boolean, you can just use the function has_angular_velocity()

Reply:
Ah that is true. I will implement that. (I honestly do not think it changes that much, but I guess it can not hurt :) )

Reply:
Well, actually... So I tried to use the suggested optionality, but proto3 does not support optional fields. There is no proper way to test if a scalar (floats, bools, etc) has been set, or simply is 0. There is the possibility to do some trickery, like with oneof (either float or a bool, where bool (false or true) means it has not been set), or using experimental code of protobuf that does add the `required` tag, but all these options makes the code unnecessarily more complex for no real reason. Efficiency wise, this does not matter anyways

Reply:
I believe proto3 now also supports optional fields (although perhaps experimentally?). That being said, I think the way it is done now is fine as well

Reply:
![image](https://user-images.githubusercontent.com/5947757/159233196-f2bd18d6-ab90-4b4c-add3-4f934bcace74.png)
@Florisheinen1 I do not understand how the new structs fix this problem. Just as with proto, I can't tell if this float has been set or simply has the default value.

Reply:
@emielsteerneman, valid point. Floris is making some changes now to improve this. The plan is to have the field KickType be {NOTHING, KICK, CHIP} (defaulting to NOTHING). In the AI, this field is then set to either KICK or CHIP whenever we want to do either of those. When the RobotCommand is converted to a REM_RobotCommand, this can be used to set the doKick and doChip fields. So even when you can not tell whether the kickSpeed is set or not, you know whether the robot needs to kick/chip based on those fields. Does that make sense?

Reply:
@alexander-de-ranitz This  doesn't make sense to me because this is already how it's done with the current protobuf implementation. So yes, we can know if `kickSpeed(this PR)`/`chip_kick_vel(proto)` must be used by looking at the `KickType(this PR)`/`kicker(proto)`/`chipper(proto)` fields. Nothing has changed or is solved. 

The issue that @Florisheinen1 brought up with the current protobuf implementation `There is no proper way to test if a scalar (floats, bools, etc) has been set, or simply is 0.` does also not seem to be solved with this PR.

Reply:
@emielsteerneman You're right in saying that nothing really changes or is improved in terms of what information the proto packets contain- it is functionally completely the same. However, it is not the intention of this PR to improve/change this (afaik). The purpose of this PR is to create a struct that can be easily converted to/from proto messages so that we have to use the actual proto messages as little as possible in our code. Structs are much easier/nicer to use throughout our code as proto messages can be hard to read, contain weird/unreadable data, etc.